File: examples/comparison/struct_sum/StructSum.java
public class StructSum {
    static class Point {
        int x;
        int y;
    }

    public static void main(String[] args) {
        long sum = 0;
        final int n = 80_000_000;
        Point p = new Point();
        p.x = 1;
        p.y = 2;
        for (int i = 0; i < n; i++) {
            sum += p.x + p.y;
        }
        System.out.println(sum);
    }
}

File: examples/comparison/struct_sum/struct_sum.go
package main

import "fmt"

type Point struct {
	x int
	y int
}

func main() {
	var sum int64 = 0
	const n = 80_000_000
	p := Point{x: 1, y: 2}
	for i := 0; i < n; i++ {
		sum += int64(p.x + p.y)
	}
	fmt.Println(sum)
}
